src/libstore/meson.build

```diff
@@ -352,13 +352,7 @@ libraries_private = []
 
 extra_pkg_config_variables = {
   'storedir' : get_option('store-dir'),
+  'localstatedir' : get_option('localstatedir'),
```

Comment:
Do we need to increase the minimum required version of meson?
There are other Linux distributions that build nix from source.

Reply:
Yes we will, and I am not sure what versions this is released in / backported to yet either.